src/FormatNumber/Parser.elm

```diff
@@ -106,6 +106,41 @@ splitThousands integers =
         |> reversedSplitThousands
         |> List.reverse
 
+{-| Split a `String` in `List String` grouping by digits as per Indian 
+numbering system. Last 3 digits are grouped together but after that 
+numbers are grouped in two. 
+[Indian numbering system](https://en.wikipedia.org/wiki/Indian_numbering_system#Use_of_separators):
+
+    splitByIndian "12345678" --> [ "1", "23", "45", "678" ]
+
+    splitByIndian "12" --> [ "12" ]
+
+-}
+splitByIndian : String -> List String
+splitByIndian integers =
+    let
+        reversedSplitHundreds : String -> List String
+        reversedSplitHundreds value =
+            if String.length value > 2 then
+                value
+                    |> String.dropRight 2
+                    |> reversedSplitHundreds
+                    |> (::) (String.right 2 value)
+
+            else if String.length value == 0 then
+                []
+            else
+                [ value ]
+        
+        thousand : String
+        thousand = 
+            if String.length integers > 3 then
+                String.right 3 integers
+            else
+                integers
+    in
+    List.reverse
+        <| thousand :: (integers |> String.dropRight 3 |> reversedSplitHundreds)
```

Comment:
What about enhancing readability by using only one direction of `|>` or `<|` in the same expression?

Not sure if this is accurate, but I was wondeirng something like:

```suggestion
    integers
        |> String.dropRight 3
        |> reversedSplitHundreds
        |> (::) thousand
        |> List.reverse
```

Reply:
Agreed. I wanted to build it similarly but could not come up with this sequence. :)